wgkex/worker/netlink.py

```diff
@@ -67,7 +67,7 @@ def wg_flush_stale_peers(domain: str) -> List[Dict]:
         The peers which we can remove.
     """
     stale_clients = [
-        stale_client for stale_client in find_stale_wireguard_clients("wg-" + domain)
+        stale_client for stale_client in find_stale_wireguard_clients("wg" + domain)
```

Comment:
we need this hyphen ... maybe also something to add to configuration like "wg_interface_prefix"?